plugins/Installation/lang/en.json

```diff
@@ -1,14 +1,14 @@
 {
     "Installation": {
-        "CollaborativeProject": "Matomo is a collaborative project, built with love by people from all over the world.",
-        "ConfigurationHelp": "Your Matomo configuration file appears to be misconfigured. You can either remove config\/config.ini.php and resume installation, or correct the database connection settings.",
-        "ConfirmDeleteExistingTables": "Are you sure you want to delete the tables: %s from your database? WARNING: DATA FROM THESE TABLES CANNOT BE RECOVERED!",
+        "CollaborativeProject": "Matomo is a collaborative project built with love.",
```

Comment:
Maybe slightly less informative, would suggest removing this change.

Reply:
Just keep it short. There are enough walls of text.
"All over the world" is as meaningless as it is incorrect. It isn't going to be _all_ over the world, and what sort of marketing is that really?